kafka/SOURCES/kafka.init

```diff
@@ -87,7 +75,6 @@ stopServiceHandler() {
   kv.sendSignal SIGTERM
```

Comment:
You remove halt command, but not implement `force` argument usage in `stopServiceHandler` function.